js/pages/characterizations/services/CharacterizationService.js

```diff
@@ -44,7 +44,16 @@ define([
     function loadCharacterizationExecutionList(id) {
         return httpService
             .doGet(config.webAPIRoot + 'cohort-characterization/' + id + '/generation')
-            .then(res => res.data);
+            .then(res => generateVersionTags(res.data));
+    }
+
+
+    function generateVersionTags(sourceInfoList) {
```

Comment:
Param name is confusing, smth like `ccGenerations` would be better.
Actually it isn't sources list.

Reply:
done